resources/js/store/actions.js

```diff
@@ -237,6 +249,10 @@ export const actions = {
             });
             media.push(response);
           })
+          // ***************************
+          .catch((err)=> {
+            console.log("move select media - err", err);
```

Comment:
instead of console logging i think we can throw an error here, [Error() constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error). they both do the same thing essentially but the error constructor is there for that reason.